### DISCUSSION TITLE
Errors using Gradle with OpenJDK@16

### DISCUSSION THREAD

**Discussion by MauriceReeves-usds** [CDCgov/prime-reportstream#1207] [General]
Brew upgraded by OpenJDK to v16 and since then Gradle has not worked for me on the command line. Intellij has an internal v11 JVM that it can use to build, and so I'm not stuck there, but none of the fixes that I've seen online have actually worked to fix this. I wanted to post this in case anyone else runs into this issue, or comes across a fix.

**Comment by MauriceReeves-usds** [comment]
I should say, none of the fixes work for running Gradle on top of openjdk@16. It does work if you set your `JAVA_HOME` to explicitly point to openjdk@11. That wasn't necessary with openjdk@15. 🤷‍♂️

**Reply by RickHawesUSDS** [reply]
Do you use `jenv` or `sdkman` to handle multiple JDK and Java versions? I've used these tools when I was working on multiple projects with different JDK requirements.

**Reply by MauriceReeves-usds** [reply]
I haven't, but I probably should start doing that.
